src/utils/lbann_library.cpp

```diff
@@ -192,6 +195,9 @@ std::unique_ptr<trainer> construct_trainer(lbann_comm *comm,
       std::cout << "\n"
                 << trainer->get_description()
                 << std::endl;
+
+      // User feedback
+      print_parameters(*comm, pb, random_seed, data_seq_random_seed);
```

Comment:
The reported seeds will be different from the user-provided seeds since they've gone through several `hash_combine`. This may confuse users who manually set the seed.

Reply:
I have improved the reporting to include the base and the actual used parameters.